qemu/tests/flow_caches_stress_test.py

```diff
@@ -83,6 +83,10 @@ def run(test, params, env):
     passwd = params.get("hostpasswd", "123456")
     client = params.get("shell_client", "ssh")
     port = params.get("shell_port", "22")
+    shell_prompt = params.get("shell_prompt")
+    linesep = params.get("shell_linesep").decode('string_escape')
+    status_test_command = params.get("status_test_command")
```

Comment:
For those params which don't have default values but they are a must in the case, please use params['shell_prompt'].